Client/Frontend/Widgets/SiteTableViewController.swift

```diff
@@ -165,10 +165,6 @@ class SiteTableViewController: UIViewController, UITableViewDelegate, UITableVie
         return cell
     }
 
-    func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
```

Comment:
viewForHeaderInSection conflicts with titleForHeaderInSection (can't have both), and we want to use the latter in a subclass. Thus, remove the former from the base class and require base classes to implement it if they need it.